DEVGUIDE.md

```diff
@@ -34,113 +34,60 @@ Install
 
-### Developing the F# Compiler (Linux)
-
-For Linux/Mono, follow [these instructions](http://www.mono-project.com/docs/getting-started/install/linux/). Also you may need:
-
-    sudo apt-get install mono-complete make git
-
-Then:
-
-    make
-
-Then to replace your machine-wide installation:
-
-    sudo make install
-
-Full testing is not yet enabled on Linux.
-
-### Developing the F# Compiler (macOS)
+### Developing the F# Compiler (Linux/macOS)
 
-Install XCode command line tools (or homebrew equivalents) and Mono or Visual Studio for Mac.
+For Linux/Mac:
 
-Then:
+    ./build.sh
 
-    make
+Running tests:
 
-Then to replace your machine-wide installation:
-
-    sudo make install
-
-Full testing is not yet enabled on macOS.
-
-### [Optional] Specifying the install path (Linux or macOS)
-
-You can specify a custom installation path using the DESTDIR shell variable
-
-    DESTDIR=/my/path/to/fsharp make install
-
-### Developing the F# Compiler (Linux or macOS - .NET Core)
-
-Install [the latest .NET SDK](https://www.microsoft.com/net/download/).  Then use
-
-    src/buildfromsource.sh
-
-Outputs are placed in
-
-    BuildFromSource/Debug/...
-    BuildFromSource/Release/...
-
-This uses an installed .NET SDK 2.0 to build the various duplicated project
-
-Testing the .NET Core version of the F# compiler on macOS and Linux is TBD.
+    ./build.sh -test
 
 ### Developing the Visual F# IDE Tools (Windows Only)
 
 To build and test Visual F# IDE Tools, install these requirements:
 
 - Download [Visual Studio 2017](https://www.visualstudio.com/downloads/)
 - Launch the Visual Studio Installer
-  - Under the "Windows" workloads, select ".NET desktop development"
-    - Select "F# desktop language support" under the optional components
-  - Under the "Other Toolsets" workloads, select "Visual Studio extension development"
+  - Under the **"Windows"** workload, select **".NET desktop development"**
+    - Select the optional component **"F# desktop language support"**
+  - Under the **"Mobile & Gaming"** workload, select **"Mobile development with .NET"**
+  - Under the **"Other Toolsets"** workload, select **"Visual Studio extension development"**
+  - On the **"Individual Components"** tab, select **".NET Framework 4.7.2 SDK"** and **".NET Framework 4.7.2 targeting pack"**
 
 Steps to build:
 
-    build.cmd vs              -- build the Visual F# IDE Tools in Release configuration (see below)
-    build.cmd vs debug        -- build the Visual F# IDE Tools in Debug configuration (see below)
-    build.cmd vs test         -- build Visual F# IDE Tools, run all tests (see below)
+    Build.cmd                             -- build all F# components under the default configuration (Debug)
+    Build.cmd -configuration Release      -- build all F# components as Release
+    Build.cmd -testDesktop                -- build and test all net46 tests
+
+All test options:
+
+    -testDesktop                          -- test all net46 target frameworks
```

Comment:
Minor nit: the capitalization in these flags looks odd - `Fcs`?  we never use that.  Just stick to lowercase for flags

Reply:
I use mixed case to try to improve readability, but Powershell is case insensitive when parsing switches so you can supply either at the command line.

Reply:
OK, then let's just change the help.  You're sort of right that it improves readability, but the whole world just expects command line flags to be lower case, and we don't want to look odd :)